mps_youtube/player.py

```diff
@@ -89,10 +89,10 @@ def play(self, songlist, shuffle=False, repeat=False, override=False):
             if config.SET_TITLE.get:
                 util.set_window_title("mpsyt")
 
+            print("songlistlen " + str(self.song_no) + ":" + str(len(songlist)))
```

Comment:
Still have this line (which presumably is debugging code not meant to be committed) and another change in this file.